inputs/prod_inner.json

```diff
@@ -124,6 +124,17 @@
       }
     },
     {
+      "name": "koji_upload",
+      "args": {
+        "kojihub": "{{KOJI_HUB}}",
+        "url": "{{OPENSHIFT_URI}}",
+        "verify_ssl": false,
+        "blocksize": 10485760,
+        "koji_keytab": false,
+        "koji_principal": false
+      }
+    },
+    {
```

Comment:
This needs to go in the worker inner JSON for the new arrangement, not in prod_inner.json, and it also needs to run after the "compress" plugin. Best is between "compress" and "pulp_pull" I think.